File: PS1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int hei;
    do
    {
        hei = get_int("Height: ");
    }
    while (hei < 1 || hei > 8);
    for (int a = 1; a <= hei; a++)
    {
        for (int c = 1; c <= hei - a; c++)
        {
            printf(" ");
        }
        for (int b = 1; b <= a; b++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: PS6/sentimental-mario-more/mario.py
# TODO
from cs50 import get_int
while 1:
    height = get_int('Height: ')
    if height > 8 or height < 1:
        print('Height is should among 1 to 8 ')
        continue
    else:
        break
for a in range(height):
    print((height - (a + 1)) * ' ' + (a + 1) * '#' + '  ' + (a + 1) * '#')